brvs-rules/src/main/java/iroha/validation/rules/impl/billing/BillingInfo.java

```diff
@@ -0,0 +1,151 @@
+package iroha.validation.rules.impl.billing;
+
+import com.google.gson.JsonObject;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class BillingInfo {
+
+  private static final DateTimeFormatter isoLocalDateTime = DateTimeFormatter.ISO_LOCAL_DATE_TIME;
+  private static final String ACCOUNT_ID_ATTRIBUTE = "accountId";
+  private static final String BILLING_TYPE_ATTRIBUTE = "billingType";
+  private static final String ASSET_ATTRIBUTE = "asset";
+  private static final String FEE_FRACTION_ATTRIBUTE = "feeFraction";
+  private static final String CREATED_ATTRIBUTE = "created";
+  private static final String UPDATED_ATTRIBUTE = "updated";
+
+  private String domain;
+  private BillingTypeEnum billingType;
+  private String asset;
+  private BigDecimal feeFraction;
+  private LocalDateTime updated;
+
+  private BillingInfo(
+      String domain,
+      BillingTypeEnum billingType,
+      String asset,
+      BigDecimal feeFraction,
+      LocalDateTime updated) {
+
+    this.domain = domain;
+    this.billingType = billingType;
+    this.asset = asset;
+    this.feeFraction = feeFraction;
+    this.updated = updated;
+  }
+
+  public enum BillingTypeEnum {
+    TRANSFER("transfer"),
+    CUSTODY("custody"),
+    ACCOUNT_CREATION("accountCreation"),
+    EXCHANGE("exchange"),
+    WITHDRAWAL("withdrawal");
+
+    public final String label;
+
+    /* default */ BillingTypeEnum(String label) {
```

Comment:
Why `/* default */`?

Reply:
To indicate `package-private` scope is put on purpose